source/Mlos.Python/mlos/OptimizerEvaluationTools/unit_tests/TestOptimizerEvaluator.py

```diff
@@ -102,58 +101,58 @@ def test_defaults(self):
         #
         with open(os.path.join(self.temp_dir, "execution_info.json")) as in_file:
             execution_info = json.load(in_file)
-        self.assertTrue(execution_info["success"])
-        self.assertTrue(execution_info["num_optimization_iterations"] == optimizer_evaluator_config.num_iterations)
-        self.assertTrue(execution_info["evaluation_frequency"] == optimizer_evaluator_config.evaluation_frequency)
-        self.assertTrue(execution_info["exception"] is None)
-        self.assertTrue(execution_info["exception_stack_trace"] is None)
+        assert execution_info["success"]
+        assert execution_info["num_optimization_iterations"] == optimizer_evaluator_config.num_iterations
+        assert execution_info["evaluation_frequency"] == optimizer_evaluator_config.evaluation_frequency
+        assert execution_info["exception"] is None
+        assert execution_info["exception_stack_trace"] is None
 
         with open(os.path.join(self.temp_dir, "execution_trace.json")) as in_file:
             trace = json.load(in_file)
-        self.assertTrue(len(trace) > 100)
-        self.assertTrue(all(key in trace[0] for key in ["ts", "name", "ph", "cat", "pid", "tid", "args"]))
+        assert len(trace) > 100
+        assert all(key in trace[0] for key in ["ts", "name", "ph", "cat", "pid", "tid", "args"])
 
         with open(os.path.join(self.temp_dir, "goodness_of_fit.pickle"), "rb") as in_file:
             unpickled_gof = pickle.load(in_file)
 
         gof_df = unpickled_gof.get_goodness_of_fit_dataframe()
-        self.assertTrue(len(gof_df.index) > 0)
-        self.assertTrue(all((col_name in gof_df.columns.values or col_name == "data_set_type") for col_name in GoodnessOfFitMetrics._fields))
-        self.assertTrue(all(
+        assert len(gof_df.index) > 0
+        assert all((col_name in gof_df.columns.values or col_name == "data_set_type") for col_name in GoodnessOfFitMetrics._fields)
+        assert (all(

```

Comment:
nit: you removed parentheses elsewhere. Is there a reason to keep this pair?

Reply:
Different regex and in general with linebreaks they might be necessary. Here they indeed can be removed.